website/docs/r/synthetics.html.markdown

```diff
@@ -125,6 +125,10 @@ The following arguments are supported:
   - `method` - (Required) no-op, use GET
   - `url` - (Required) Any url
 - `request_headers` - (Optional) Header name and value map
+- `request_query` - (Optional) Query arguments name and value map
+- `request_basicauth` - (Optional) Array of 1 item containing HTTP basic authentication credentials
```

Comment:
Why is it an array of one item and not the item directly?

Reply:
In terraform you can't enforce keys on map. Having an array of 1 item allows having a schema, and thus enforcing the 2 keys.